kong/plugins/request-transformer/schema.lua

```diff
@@ -10,8 +10,18 @@ local function check_for_value(value)
   return true
 end
 
+local function check_method(value)
+  if not value then return true end
+  local method = value:upper()
+  local ngx_method = ngx["HTTP_"..method]
+  if not ngx_method then
+    return false, method.." is not supported"
```

Comment:
But that's not how this method is being used. It's passed to `func`, and it is a common Lua idiom when the second parameter is an error to return `nil` as the first one:
```lua
local ok, err = check_method("hello")
if not ok then

end
```

Returning `false` would make sense if, say, there is no error as the 2nd return value.